app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
+    post = user.posts[params[:post_id].to_i - 1]
+    comment = Comment.new(comment_params)
+    comment.author = current_user
+    comment.post = post
```

Comment:
- [ ] for this part you created the `current_user` method, you can write it something like this:
```
    @comment = @current_user.comments.new(params)
    @comment.post_id = params[:post_id]
```